src/main/resources/cmmn/Generiek_zaakafhandelmodel.cmmn.xml

```diff
@@ -1,191 +1,216 @@
-             xmlns:flowable="http://flowable.org/cmmn" xmlns:cmmndi="http://www.omg.org/spec/CMMN/20151109/CMMNDI"
-             xmlns:dc="http://www.omg.org/spec/CMMN/20151109/DC" xmlns:di="http://www.omg.org/spec/CMMN/20151109/DI"
-             targetNamespace="http://www.flowable.org/casedef" exporter="Flowable Open Source Modeler"
-             exporterVersion="6.7.2">
+<definitions xmlns="http://www.omg.org/spec/CMMN/20151109/MODEL" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:flowable="http://flowable.org/cmmn" xmlns:cmmndi="http://www.omg.org/spec/CMMN/20151109/CMMNDI" xmlns:dc="http://www.omg.org/spec/CMMN/20151109/DC" xmlns:di="http://www.omg.org/spec/CMMN/20151109/DI" xmlns:design="http://flowable.org/design" targetNamespace="http://www.flowable.org/casedef" design:palette="flowable-work-case-palette">
     <case id="generiek-zaakafhandelmodel" name="Generiek zaakafhandelmodel" flowable:initiatorVariableName="initiator">
         <documentation>Taak "Verzenden document" toegevoegd</documentation>
-        <casePlanModel id="GENERIEK_ZAAKAFHANDELMODEL" name="Generiek zaakafhandelmodel" flowable:formFieldValidation="true"
-                       autoComplete="true">
-            <planItem id="planItem4" name="Intake" definitionRef="INTAKE">
+        <casePlanModel id="GENERIEK_ZAAKAFHANDELMODEL" name="Generiek zaakafhandelmodel" flowable:formFieldValidation="true" autoComplete="true">
+            <extensionElements>
+                <design:stencilid><![CDATA[ExpandedStage]]></design:stencilid>
+            </extensionElements>
+            <planItem id="planItemINTAKE" name="Intake" definitionRef="INTAKE">
                 <itemControl>
                     <requiredRule></requiredRule>
                 </itemControl>
             </planItem>
-            <planItem id="planItem11" name="In behandeling" definitionRef="IN_BEHANDELING">
-                <entryCriterion id="sid-563BE4E9-78ED-432A-9236-1D0708028133" sentryRef="sentry8"></entryCriterion>
+            <planItem id="planItemIN_BEHANDELING" name="In behandeling" definitionRef="IN_BEHANDELING">
+                <entryCriterion id="sid-563BE4E9-78ED-432A-9236-1D0708028133" sentryRef="sentrysid-563BE4E9-78ED-432A-9236-1D0708028133"></entryCriterion>
             </planItem>
-            <sentry id="sentry8">
-                <planItemOnPart id="sentryOnPart3" sourceRef="planItem4">
+            <sentry id="sentrysid-563BE4E9-78ED-432A-9236-1D0708028133">
+                <extensionElements>
+                    <design:stencilid><![CDATA[EntryCriterion]]></design:stencilid>
+                </extensionElements>
+                <planItemOnPart id="sentryOnPartsid-563BE4E9-78ED-432A-9236-1D0708028133" sourceRef="planItemINTAKE">
                     <standardEvent>complete</standardEvent>
                 </planItemOnPart>
-                <ifPart>
+                <ifPart id="sentryIfPart_sid-563BE4E9-78ED-432A-9236-1D0708028133">
```

Comment:
Is this an id specifically added for Sentry? Why?

Reply:
I have no idea. :-) I just copied this XML from the export of the Flowable Cloud editor. My assumption is that the Flowable Cloud editor is the only place where we will ever manage our CMMN model..